fdbrpc/SimExternalConnection.actor.cpp

```diff
@@ -67,8 +67,11 @@ class SimExternalConnectionImpl {
 	}
 };
 
+std::map<std::string, std::vector<NetworkAddress>> SimExternalConnection::hostnameToAddresses;
+
 void SimExternalConnection::close() {
 	socket.close();
+	clearMockTCPEndpoints();
```

Comment:
`close` is a non-static method. This indicates that closing one connection would clear the entire hostnameToAddresses map. Is this intended?

Reply:
No, I was doing wrong.